core/iwasm/interpreter/wasm_loader.c

```diff
@@ -713,34 +712,8 @@ wasm_loader_find_export(const WASMModule *module, const char *module_name,
         return NULL;
     }
 
-    switch (export_kind) {
-        case EXPORT_KIND_FUNC:
-            export_index_boundary =
-                module->import_function_count + module->function_count;
-            break;
-        case EXPORT_KIND_GLOBAL:
-            export_index_boundary =
-                module->import_global_count + module->global_count;
-            break;
-        case EXPORT_KIND_MEMORY:
-            export_index_boundary =
-                module->import_memory_count + module->memory_count;
-            break;
-        case EXPORT_KIND_TABLE:
-            export_index_boundary =
-                module->import_table_count + module->table_count;
-            break;
-        default:
-            bh_assert(0);
-    }
-
-    if (export->index >= export_index_boundary) {
-        LOG_DEBUG("%s in the module %s is out of index (%d >= %d )", field_name,
-                  module_name, export->index, export_index_boundary);
-        set_error_buf(error_buf, error_buf_size, "incompatible import type");
-        return NULL;
-    }
-
+    /* since there is a validation in load_export_section(), it is for sure
+     * export->index is valid*/
```

Comment:
The export index is checked in wasm_loader.c load_export_section, but isn't checked in aot_loader.c load_export_section, shall we check it?
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/aot/aot_loader.c#L1765-L1772